spring-auto-restdocs-core/src/main/java/capital/scalable/restdocs/response/ResponseModifyingPreprocessors.java

```diff
@@ -20,10 +20,7 @@
 package capital.scalable.restdocs.response;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
-import org.springframework.restdocs.operation.preprocess.ContentModifyingOperationPreprocessor;
-import org.springframework.restdocs.operation.preprocess.OperationPreprocessor;
-import org.springframework.restdocs.operation.preprocess.OperationResponsePreprocessor;
-import org.springframework.restdocs.operation.preprocess.Preprocessors;
+import org.springframework.restdocs.operation.preprocess.*;
```

Comment:
Please expand the imports again. We are avoiding `*` imports.  But I also know that we neither have this documented nor enforced at the moment.

Reply:
Fixed